packages/ckeditor5-fullscreen/src/handlers/abstracteditorhandler.ts

```diff
@@ -19,7 +19,49 @@ import type { Annotation, AnnotationsUIs, Sidebar } from '@ckeditor/ckeditor5-co
 const DIALOG_OFFSET = 28;
 
 /**
- * The abstract editor type handler. It should be extended by the particular editor type handler.
+ * The abstract editor type handler.
+ *
+ * When entering the fullscreen mode, the `#enable()` method is called. It creates the properly styled container
+ * and handles the editor features that need it, in the following order:
+ *
+ * 1. Save the scroll positions of all ancestors of the editable element to restore them after leaving the fullscreen mode.
+ * 2. Execute the `#_defaultOnEnter()` method to move the proper editor UI elements to the fullscreen mode. **If you extend the abstract
+ * 	handler, you should override this method** to move the elements that are specific to your editor type, like:
+ * 	editable, toolbar, menu bar.
+ * 	Use `#moveToFullscreen()` method for this purpose to ensure they are automatically cleaned up after leaving the fullscreen mode.
+ * 3. Add proper classes to the <body> and <html> elements to block page scrolling, adjust z-index etc.
+ * 4. Change the position of some dialogs to utilize the empty space on the right side of the editable element.
+ *
+ * Steps 5-11 are only executed if the corresponding features are used.
+ *
+ * 5. If presence list is used, move it to the fullscreen mode.
+ * 6. If document outline is used, move it to the fullscreen mode.
+ * 7. If pagination is used, adjust it's configuration for the changed view.
+ * 8. If annotations are used, move them to the fullscreen mode.
+ * 9. If revision history is used, override the callbacks to show the revision viewer in the fullscreen mode.
+ * 10. If source editing and document outline are both used, hide the document outline header.
+ * 11. If custom container is used, hide all other elements in it to ensure they don't create an empty unscrollable space.
+ *
+ * Then finally:
+ *
+ * 12. Execute the configured `config.fullscreen.onEnterCallback` function.
```

Comment:
Does this function have any default? If so, what the default does? If there's no default, I'd rewrite this to "Execute `config.fullscreen.onEnterCallback()` function, if set."